bc_obps/reporting/json_schemas/2024/pulp_and_paper_production/pulp_and_paper_production.json

```diff
@@ -0,0 +1,53 @@
+{
+  "type": "object",
+  "title": "Pulping and chemical recovery",
+  "properties": {
+    "emissions": {
+      "type": "array",
+      "title": "Emissions",
+      "items": {
+        "title": "Pulping and chemical recovery",
+        "type": "object",
+        "properties": {
+          "carbonateName": {
+            "type": "string",
+            "title": "Carbonate Name"
+          },
+          "annualAmount": {
+            "type": "number",
+            "title": "Annual Amount (t)"
+          },
+          "purityOfCarbonate": {
+            "type": "number",
+            "title": "Purity Of Carbonate (Weight Fraction)"
+          },
+          "methodology": {
+            "title": "Methodology",
+            "type": "string",
+            "enum": [
+              "Solids-HHV",
+              "Solids-CC",
+              "Make-up Chemical Use Methodology",
+              "Alternative Parameter Methodology",
+              "Replacement Methodology"
+            ]
+          },
```

Comment:
This methodology object is unnecessary here, it gets dynamically added by the form builder service via the configurations in the database
```suggestion
```

Reply:
Fixed with 46fcf40.